addons/website/static/src/snippets/s_process_steps/options.js

```diff
@@ -18,6 +18,16 @@ options.registry.StepsConnector = options.Class.extend({
         this._super(...arguments);
         this.$target.off('.StepsConnector');
     },
+    /**
+     * @override
+     */
+    async updateUI() {
+        if (this.options.editable[0].classList.contains('o_is_mobile')) {
```

Comment:
I think there is a prettier way to check mobile mode is enabled. Check the DeviceVisibility option, I think there is a trigger_up to do it.